packages/x-scheduler/src/internals/components/event-popover/EventPopover.tsx

```diff
@@ -110,6 +110,8 @@ export const EventPopoverContent = React.forwardRef(function EventPopoverContent
     store.setOccurrencePlaceholder({
       type: 'creation',
       surfaceType: surfaceTypeToUse,
+      // TODO: Pass the right resourceId
```

Comment:
I'm waiting for @noraleonte refacto before doing this change.
It will allow the created event to move rows on the timeline when changing the resource in the UI.